samples/tools/testbed/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Host a jsPsych experiment in Azure
+FROM python:3.11
+MAINTAINER AutoGen
+
+# Upgrade pip
+RUN pip install --upgrade pip
+
+# Set the image to the Pacific Timezone
+RUN ln -snf /usr/share/zoneinfo/US/Pacific /etc/localtime && echo "US/Pacific" > /etc/timezone
+
+# Pre-load autogen dependencies, but not autogen itself since we'll often want to install the latest from source
+RUN pip install pyautogen[teachable]
```

Comment:
Why is this pyautogen[teachable] instead of lmm, retrievechat, etc?

Reply:
Retrieve chat is about 800MB of extra dependencies. We can do it... but I wasn't ready to pull that trigger yet.

Reply:
I see. Thanks for the clarification!